### PR TITLE
tag's label can accept number

### PR DESCRIPTION
**Pull request by l3satwik**

### CODE REVIEW

File: src/components/__tests__/__snapshots__/sidebar.test.tsx.snap [hunk 1]
```diff
@@ -447,7 +447,7 @@ Array [
     className="emotion-0"
     style={
       Object {
-        "opacity": 0,
+        "opacity": 0.00028516048692,
```
Comment (l3satwik) [inline src/components/__tests__/__snapshots__/sidebar.test.tsx.snap]: 🤨

Comment (azizhk) [reply src/components/__tests__/__snapshots__/sidebar.test.tsx.snap]: Tests failed over here.
These tests seem flaky as the Animation starts and slow / fast machines might have different output.

Comment (ritz078) [reply src/components/__tests__/__snapshots__/sidebar.test.tsx.snap]: better wait for some time and create a snapshot.

Comment (ritz078) [reply src/components/__tests__/__snapshots__/sidebar.test.tsx.snap]: https://github.com/ritz078/pebble/blob/23a1c815697e098489c1ea96f0b3b1ee67d6bc8f/src/components/__tests__/toast.test.tsx#L8

**Pull request merged**